readthedocs/starterguides/developerguides/object_lifecycle_management_developerguide.md

```diff
@@ -0,0 +1,81 @@
+# Developer Guide: Object Lifecycle Management
+# Overview
+This guide is intended for Object Lifecycle feature developers. It provides detailed steps to write new API using OpenSDS. To understand the feature and requirements in detail, please refer to the Object Lifecycle Management user guide and Object Lifecycle design doc. 
+
+# APIs
+APIs are a set of functions and procedures that allow for the creation of applications that access data and features of other applications, services or operating system. In OpenSDS lifecycle management feature we are using APIs to communicate with different storage backend.
+
+# Write new API
+Before starting to write the APIs, it is required to have basic understanding of XML and JSON syntax.
+
+## How to write the APIs
+
+### There are three major steps involved:
+    a) All APIs goes into the 's3' module of the OpenSDS multi-cloud package, i,e. opensds/multi-cloud/api/pkg/s3 directory
+    b) The corresponding router for the API are defined same directory.
```

Comment:
> 
```suggestion
    b) The corresponding router for the API are defined **in the** same directory.
```

Reply:
It's not used to create new API, so this chapter, Write new API, is not necessary.
As an alternative, for secondary development, you can provide the examples of creating lifecycle rule, deleting lifecycle rule, disable lifecycle rule, etc.